libr/io/plugin.c

```diff
@@ -112,6 +112,9 @@ R_API int r_io_plugin_list_json(RIO *io) {
 		io->cb_printf ("%s{\"Permissions\":\"%s\",\"Name\":\"%s\",\"Description\":\"%s\",\"License\":\"%s\"",
 				n? "," : "", str, plugin->name,
 			plugin->desc, plugin->license);
+		if (plugin->uris) {
```

Comment:
Can you update this by using pj.c library instead?

Reply:
Sure, will use pc.j library. I will follow this up on this on #13013.

Reply:
Don't close your PR next time, just push -f the changes, otherwise it's a mega-mess

Reply:
Okay @Maijin, I will definitely keep that in mind. Thanks for the feedback.

I added the pj.c library to it and pushed.